net/MapViewController.m

```diff
@@ -271,6 +276,27 @@ -(BOOL)mapView:(GMSMapView *)mapView didTapMarker:(GMSMarker *)marker
         
     }];
     self.currentMarker = marker;
+    
```

Comment:
Why do you need all this new code? Just set new issue in to CurrentItems and let the singleton make all other knew about this change